packages/contracts/contracts/foundry/Sphinx.sol

```diff
@@ -141,17 +141,24 @@ abstract contract Sphinx {
 
     function sphinxCollectProposal(
         bytes memory _scriptFunctionCalldata,
-        string memory _deploymentInfoPath
-    ) external {
+        string memory _deploymentInfoPath,
+        uint64 _callDepth
```

Comment:
We now pass the call depth as an input variable because this allows us to call `sphinxCollectProposal` in Forge tests, which is how I tested this feature. This is necessary in Forge tests because the default call depth is 3 instead of 2, since the Forge test adds one layer to the depth